week1/react-exercises/src/2-guarantee/Guarantee.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+const Guarantee = () => {
+  const guarantees = [
+    {
+      img: "./images/f-delivery.png",
+      title: "Free shipping",
+      description:
+        "Lorem, ipsum dolor sit amet consectetur adipisicing elit. Similique sunt praesentium, quaerat voluptatum consequuntur nam",
+    },
+    {
+      img: "./images/coin.png",
+      title: "100% Money back",
+      description:
+        "Lorem, ipsum dolor sit amet consectetur adipisicing elit. Similique sunt praesentium, quaerat voluptatum consequuntur nam",
+    },
+    {
+      img: "./images/chat.png",
+      title: "Online support 24/7",
+      description:
+        "Lorem, ipsum dolor sit amet consectetur adipisicing elit. Similique sunt praesentium, quaerat voluptatum consequuntur nam",
+    },
+  ];
+
+  return (
+    <div className="gContainer">
+      {guarantees.map((guar, i) => (
+        <GuaranteeLayout
+          key={i}
+          img={guar.img}
+          title={guar.title}
+          desc={guar.description}
+        />
+      ))}
+    </div>
+  );
+};
+
+const GuaranteeLayout = props => {
```

Comment:
Create a separate file for this component this way the project is more manageable.

```suggestion
const Guarantee = props => {
```

Reply:
All components are now in their own files :)